docs/how_to/publish.rst

```diff
@@ -187,6 +187,18 @@ The image needs to be created and then published:
   awspub create config.yaml
   awspub publish config.yaml
 
+Sharing images
+~~~~~~~~~~~~~~
+
+Images can be shared with other AWS accounts. For that, the account ID is required.
+
+.. literalinclude:: ../config-samples/config-minimal-share.yaml
+   :language: yaml
+
+In that example, the image `my-custom-image` will be share with the account `1234567890123`
+when `awspub` runs in the commercial partition (`aws`, the default) and will only be shared
+with the account `456789012345` when `awspub` runs in the the china partition (`aws-cn`).
```

Comment:
```suggestion
with the account `456789012345` when `awspub` runs in the the china partition (``aws-cn``).
```
By putting aws-cn in double back-ticks you can exclude it from the spelling check.